pack/parallel/enc_encounter.json

```diff
@@ -0,0 +1,79 @@
+[
+  {
+    "back_text": "Hard / Expert\n[skull]: -X. X is 1 more than the number of measures.\n[cultist]: -3. If you fail, you must spend 1 clue and take 1 damage.\n[tablet]: -4. (-6 instead if there are 3 or more classes among cards you control).\n[elder_thing]: -3. If Sinister Soloist is at your location, you automatically fail instead.",
+    "code": "90094",
+    "double_sided": true,
+    "encounter_code": "enthralling_encore",
+    "encounter_position": 1,
+    "faction_code": "mythos",
+    "name": "Entrhalling Encore",
+    "pack_code": "enc",
+    "position": 8,
+    "quantity": 1,
+    "text": "Easy / Standard\n[skull]: -X. X is the number of measures.\n[cultist]: -2. If you fail, you must either spend 1 clue or take 1 damage.\n[tablet]: -3. (-5 instead if there are 3 or more classes among cards you control).\n[elder_thing]: -2. If Sinister Soloist is at your location, reveal another token.",
+    "type_code": "scenario"
+  },
+  {
+    "back_flavor": "The soloist haunts your steps, ever present and always playing that maddening song. You feel the musical notes take root in your mind, threatening to pull you out of the theatre and into another world, one you think you've visited before...",
+    "back_name": "Dirge of Dim Carcosa",
+    "back_text": "Place 1 resource on the scenario reference card, as a measure. Then if there are 5 or more measures, each investigator is defeated and suffers 1 mental trauma.\nOtherwise, each investigator must test [willpower](3). Each investigator who fails takes 1 horror and sets aside 1 non-permanent, non-weakness asset they control. Then, shuffle each asset set aside by this effect into the Props deck. If all investigators fail this test, remove 1[per_investigator] damage from Sinister Soloist.\nPlace cluse on each revealed location, up to its clue value.\nShuffle the encounter discard pile into the encounter deck. Reset this agenda to agenda 1a.",
+    "code": "90095",
+    "doom": 4,
+    "double_sided": true,
+    "encounter_code": "enthralling_encore",
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "flavor": "The theatre is closed and quiet, as it has been since those days during the phantom production of <i>The King in Yellow</i>. As you take in your surroundings, you hear it again - the haunting violin. Someone is here, and they are keeping the memory of that priduction alive.",
```

Comment:
```suggestion
    "flavor": "The theatre is closed and quiet, as it has been since those days during the phantom production of <i>The King in Yellow</i>. As you take in your surroundings, you hear it again - the haunting violin. Someone is here, and they are keeping the memory of that production alive.",
```